### ISSUE TITLE
Does not work with Google.com in Firefox

### ISSUE THREAD

**Issue by ikkyusan123** [wong2/chatgpt-google-extension#380]
**Describe the bug**
The extension does not work with Google.com,It always displays: 'Hello! How can I assist you today?'
**Desktop (please complete the following information):**
 - OS: [ Windows 10]
 - Browser [Firefox 113]

**Comment by geek111** [comment]
Yep. I confirm, have same issue in Windows 11 and latest stable FF.
It seems it is something wrong with this addon.

**Comment by atalamcom** [comment]
yeah same... apparently in other websites work e.g. yandex..
